src/Raven.Studio/typescript/viewmodels/database/status/ongoingTasksStats.ts

```diff
@@ -1663,6 +1664,7 @@ class ongoingTasksStats extends viewModelBase {
                 if (dx >= 0.8) { // Don't show tooltip for very small items
                     this.hitTest.registerTrackItem(currentX, yStart, dx, ongoingTasksStats.trackHeight, perfItemWithCache, op);
                 }
+                this.drawTextOnStripe(context, op.Name, dx, currentX, yStart);
```

Comment:
let's move this inside `if (dx >= 0.8)` (and update comment). 

This is hot path and let's avoid calling function here when not needed.

Reply:
done